packages/jsii-pacmak/test/generated-code/__snapshots__/target-java.test.js.snap

```diff
@@ -9787,6 +9787,14 @@ public class DocumentedClass extends software.amazon.jsii.JsiiObject {
     public void hola() {
         software.amazon.jsii.Kernel.call(this, "hola", software.amazon.jsii.NativeType.VOID);
     }
+
+    /**
+     * (experimental) Say Moin, which is German for "Good morning" *&#47; (this comment is escaped because it contains a star slash).
```

Comment:
This seems to indicate your new code path was not exercised...

Reply:
Seems to be solved though, weird. I'll double check with the exact input I got.